hypothesis-python/tests/cover/test_type_lookup.py

```diff
@@ -329,9 +337,11 @@ def test_generic_origin_with_type_args(generic, strategy):
         Callable,
         List,
         Sequence,
+        tuple,
         # you can register types with all generic parameters
         List[T],
         Sequence[T],
+        tuple[T],
```

Comment:
`from typing import Tuple` here; and it'd be worth testing both `Tuple[T]` and `Tuple[T, ...]`.

We may need to wrap it in `pytest.param(Tuple[T], marks=pytest.mark.xfail(sys.version_info[:2] <= (3, 9), reason="stdlib changes")` to fix the other CI failures though, because it's treated as a parametrized generic type on those older versions.